### DISCUSSION TITLE
Chia wallet update

### DISCUSSION THREAD

**Discussion by robertbigk** [guydavis/machinaris#149] [Q&A] [answered]
Hi I use Machinaris and cant find information on how to update chcia client inside I got 1.17 now is 1.2 and pool support . Ho to update wallet for pools?

**Comment by guydavis** [comment, marked as answer]
Hi!  No need to perform updates inside the container as they won't persist beyond a restart anyway.  

As per [this announcement](https://github.com/guydavis/machinaris/discussions/153), just pull the [latest test image](https://github.com/guydavis/machinaris/wiki/Releases#release-streams) to get Machinaris with Chia 1.2.0 bundled.
